src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -52,7 +52,7 @@ describe('ReactDOMComponent', function() {
       var stubStyle = container.firstChild.style;
 
       // set initial style
-      var setup = {display: 'block', left: '1', top: 2, fontFamily: 'Arial'};
+      var setup = {display: 'block', left: '1px', top: 2, fontFamily: 'Arial'};
```

Comment:
Styles may be unitless if they are integers, but unitless strings are deprecated.  This looks fine to me.